bindata/network/openshift-sdn-azure/azure-drop-icmp-daemonset-sdn.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: add-iptables
+  namespace: openshift-sdn
+data:
+  add_iptables.sh: |
+   #!/bin/sh
+   echo "Adding ICMP drop rule for '$2' "
+   if iptables -C CHECK_ICMP_SOURCE -p icmp -s $2 -j ICMP_ACTION
+   then
+        echo "iptables already set for $2"
+   else
+        iptables -A CHECK_ICMP_SOURCE -p icmp -s $2 -j ICMP_ACTION
+   fi
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: drop-icmp
+  namespace: openshift-sdn
+  annotations:
+    kubernetes.io/description: |
+      This daemonset configures each node host to drop ICMP packets to its external address.
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+spec:
+  selector:
+    matchLabels:
+      app: drop-icmp
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      # by default, Deployments spin up the new pod before terminating the old one
+      # but we don't want that - because ovsdb holds the lock.
+      maxSurge: 0
+      maxUnavailable: 10%
+  template:
+    metadata:
+      labels:
+        app: drop-icmp
+        component: network
+        type: infra
+        openshift.io/component: network
+        kubernetes.io/os: "linux"
+    spec:
+      serviceAccountName: sdn-controller
+      hostNetwork: true
+      priorityClassName: "system-cluster-critical"
+      containers:
+      # ovnkube master: convert kubernetes objects in to nbdb logical network components
+      - name: drop-icmp
+        image: "{{.SDNImage}}"
+        command:
+        - /bin/bash
+        - -c
+        - |
+          set -xe
+          if [[ -f "/env/_master" ]]; then
+            set -o allexport
+            source "/env/_master"
+            set +o allexport
+          fi
+
+          echo "I$(date "+%m%d %H:%M:%S.%N") - drop-icmp - start drop-icmp ${K8S_NODE}"
+          iptables -X CHECK_ICMP_SOURCE || true
+          iptables -N CHECK_ICMP_SOURCE || true
+          iptables -F CHECK_ICMP_SOURCE
+          iptables -D INPUT -p icmp --icmp-type fragmentation-needed -j CHECK_ICMP_SOURCE || true
+          iptables -I INPUT -p icmp --icmp-type fragmentation-needed -j CHECK_ICMP_SOURCE
+          iptables -N ICMP_ACTION || true
+          iptables -F ICMP_ACTION
+          iptables -A ICMP_ACTION -j LOG
+          iptables -A ICMP_ACTION -j DROP
+          oc observe nodes -a '{ .status.addresses[1].address }' -- /tmp/add_iptables.sh
+        lifecycle:
+          preStop:
+            exec:
+              command: ["/bin/bash", "-c", "echo drop-icmp done"]
+        securityContext:
+          privileged: true
```

Comment:
in the interest of having fewer privileged pods, I think this only needs to be `CAP_NET_ADMIN`, not fully privileged?